addons/zylann.hterrain/hterrain_detail_layer.gd

```diff
@@ -81,6 +80,8 @@ var _debug_wirecube_mesh: Mesh = null
 var _debug_cubes := []
 var _logger := Logger.get_for(self)
 
+var _chunk_size: int = load("res://addons/zylann.hterrain/hterrain_detailer.gd").CHUNK_SIZE
```

Comment:
Perhaps the constant could actually be in that script instead.

Reply:
Do you mean define CHUNK_SIZE in both script, or define it in the detail_layer.gd and then reference it from other scripts if needed?

Reply:
@Zylann I updated it as putting CHUNK_SIZE in both script. Is this okay with you?
If so, then this PR is ready to be merged🙏